components/public-api-server/pkg/oidc/service.go

```diff
@@ -277,14 +277,16 @@ func (s *Service) Authenticate(ctx context.Context, params AuthenticateParams) (
 	}, nil
 }
 
-func (s *Service) CreateSession(ctx context.Context, flowResult *AuthFlowResult, organizationId string) (*http.Cookie, error) {
+func (s *Service) CreateSession(ctx context.Context, flowResult *AuthFlowResult, clientConfig *ClientConfig) (*http.Cookie, error) {
```

Comment:
Did we not need to update any tests to make this pass?

Reply:
Added test for CreateSession. 
Next time we would.